examples/tutorials/tutorial_udd_forces_and_moments.py

```diff
@@ -0,0 +1,177 @@
+        outputs=[
+            fl.VolumeOutput(
+                output_fields=["primitiveVars", "vorticity", "qcriterion", "Cp", "Mach"]
+            ),
+            fl.SurfaceOutput(
+                surfaces=[geometry["*"]],
+                output_fields=["primitiveVars", "Cf", "wallDistance", "Cp", "CfVec", "yPlus"],
+            ),
+        ],
+        user_defined_dynamics=[
+            fl.UserDefinedDynamic(
+                name="rightAileronHingeTorque",
+                input_vars=["forceX", "forceY", "forceZ", "momentX", "momentY", "momentZ"],
+                constants={
+                    "density_kgpm3": 1.225,
+                    "c_inf_mps": 340.29400580821283,
+                    "l_grid_unit": 1,
+                    "newCenterX": 5.7542,
+                    "newCenterY": 7,
+                    "newCenterZ": 0,
+                    "newAxisX": 0,
+                    "newAxisY": 1,
+                    "newAxisZ": 0,
+                },
+                state_vars_initial_value=["0.0", "0.0", "0.0", "0.0", "0.0"],
+                update_law=[
+                    "density_kgpm3 * c_inf_mps * c_inf_mps * l_grid_unit * l_grid_unit * l_grid_unit;",
+                    "(momentX - ((newCenterY - momentCenterY) * forceZ - (newCenterZ - momentCenterZ) * forceY)) * state [0];",
+                    "(momentY + ((newCenterX - momentCenterX) * forceZ - (newCenterZ - momentCenterZ) * forceX)) * state [0];",
+                    "(momentZ - ((newCenterX - momentCenterX) * forceY - (newCenterY - momentCenterY) * forceX)) * state [0];",
+                    "state[1] * newAxisX + state[2] * newAxisY + state[3] * newAxisZ;",
+                ],
+                input_boundary_patches=[geometry["aileronRight"]],
+            ),
+            fl.UserDefinedDynamic(
+                name="leftAileronHingeTorque",
+                input_vars=["forceX", "forceY", "forceZ", "momentX", "momentY", "momentZ"],
+                constants={
+                    "density_kgpm3": 1.225,
+                    "c_inf_mps": 340.29400580821283,
+                    "l_grid_unit": 1,
+                    "newCenterX": 5.7542,
+                    "newCenterY": -7,
+                    "newCenterZ": 0,
+                    "newAxisX": 0,
+                    "newAxisY": -1,
+                    "newAxisZ": 0,
+                },
+                state_vars_initial_value=["0.0", "0.0", "0.0", "0.0", "0.0"],
+                update_law=[
+                    "density_kgpm3 * c_inf_mps * c_inf_mps * l_grid_unit * l_grid_unit * l_grid_unit",
+                    "(momentX - ((newCenterY - momentCenterY) * forceZ - (newCenterZ - momentCenterZ) * forceY)) * state [0];",
+                    "(momentY + ((newCenterX - momentCenterX) * forceZ - (newCenterZ - momentCenterZ) * forceX)) * state [0];",
+                    "(momentZ - ((newCenterX - momentCenterX) * forceY - (newCenterY - momentCenterY) * forceX)) * state [0];",
```

Comment:
Can you remove the space between `state` and `[0]` just to keep the format consistent? Please also check other places.